src/miniapps/CMakeLists.txt

```diff
@@ -0,0 +1,28 @@
+PROJECT(miniapps)
```

Comment:
They are harmless. Not build by default. I also use this place to host restart I/O miniapps.